docs/.vuepress/theme/SearchBox.vue

```diff
@@ -63,8 +63,8 @@ export default {
       for (let i = 0; i < pages.length; i++) {
         if (res.length >= max) break
         const p = pages[i]
-        // filter out results that do not match current ersion context
-        if (~p.path.slice(1).indexOf(searchContext)) continue
+        // filter out results that do not match current version context
+        if (!~p.path.slice(1).indexOf(searchContext)) continue
```

Comment:
Can we use `includes`?; https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes

Or, even better, `startsWith`: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith

If, I'm understanding well:
```suggestion
         if ( ! p.path.startsWith(searchContext, 1)) {
           continue;
         }
``` 

It is more descriptive and we avoid use that "hacky" `!~`, isn't?.

Reply:
Makes sense! Thank you. It should be properly transpiled for older browsers.

`!~` isn't hacky, but not really descriptive :)
I would very much favor your solution